selfdrive/car/tests/test_docs.py

```diff
@@ -74,8 +74,11 @@ def test_harnesses(self):
         if car.name == "comma body":
           raise unittest.SkipTest
 
-        self.assertNotIn(car.harness_kit.connector, [None, Harness.none], f"Need to specify car harness: {car.name}")
-        self.assertTrue(car.harness_kit.parts, f"Need to specify harness parts: {car.name} with {car.harness_kit.connector.value} connector")
+        car_part_type = [p.value.type for p in car.car_parts.parts]
+        self.assertTrue(len(car.car_parts.parts) > 0, f"Need to specify car parts: {car.name}")
+        self.assertTrue(car_part_type.count(PartType.connector) == 1, f"Need to specify one harness connector: {car.name}")
+        self.assertTrue(car_part_type.count(PartType.mount) == 1, f"Need to specify one mount: {car.name}")
+        self.assertTrue(CarPart.right_angle_obd_c_cable_1_5ft in car.car_parts.parts, f"Need to specify a right angle OBD-C cable (1.5ft): {car.name}")
```

Comment:
if there's any other common parts you can check them here too, else just close this

Reply:
the intersection for the moment is only a mount and a connector (unless we add the comma3 or a panda in the parts)

Reply:
should add comma three too (can do it in the next PR)